up

```diff
@@ -46,7 +46,6 @@ setuphosts() {
 }
 
 rundocker() {
-    # sudo -E docker-compose scale mender-client=3
     sudo -E ./demo up
```

Comment:
I think it's easier to set an environment variable stating how many clients you want instead of knowing the scale argument of docker-compose.